### PR TITLE
Support our Windows-based friends

### PR DESCRIPTION
**Pull request by d3r3kk**
Added some documentation changes that help direct Windows dev workflow specifically.

### CODE REVIEW

**Comment by d3r3kk** [comment]
(Also took some liberties with removing unused imports from `gamatrix-gog.py`)

**Comment by d3r3kk** [comment]
Should satisfy #23 without resorting to a Windows-based container I hope.

File: config-sample-windows.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# The GOG user ID matching the result of 'select * from Users;' in the DB
+users:
+  12345:
+    username: Bob
+    db: bob-galaxy-2.0.db
+  56789:
+    username: Doug
+    db: doug-galaxy-2.0.db
+metadata:
+  # Each title must match exactly as listed with --all-games; normal YAML quoting rules apply
```
Comment (eniklas) [inline config-sample-windows.yaml:33]: I need to update this; the match now happens on the sanitized title so this isn't really true, but it's still a good idea to put it exactly as listed by `--all-games`.

File: gamatrix-gog.py [hunk 2]
```diff
@@ -1,14 +1,9 @@
...
 import argparse
-import copy
 from helpers.cache_helper import Cache
-import json
 import logging
 import os
-import re
-import sqlite3
 import sys
-import time
```
Comment (eniklas) [inline gamatrix-gog.py:11]: Nice, my linter must have been broken when I moved stuff into modules 🤦

File: README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## Quick start
 
-Jump to [command-line mode](#cli) or [building with Docker](#docker).
+Jump to [command-line mode](#command-line-mode) or [building with Docker](#running-in-docker).
```
Comment (eniklas) [inline README.md:5]: Oh nice, so there's a default anchor that matches the heading text? Never knew that and always hated embedding HTML anchors.

**Review by eniklas — APPROVED** [review]
Good stuff!

**Pull request merged**